src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -162,15 +162,15 @@ private String getRequestContentLines(Map<String, Object> contents) {
             if (entry.getKey().equals(PARAMS_KEY))
                 continue;
   
-            String line = "<strong>" + customizeKey(entry.getKey()) + ":</strong>" + entry.getValue().toString() + "<br>";
+            String line = "<tr><th role=\"rowheader\" scope=\"row\">" + customizeKey(entry.getKey()) + ":</th><td>" + entry.getValue().toString() + "</td></tr>";
```

Comment:
@hsong-rh Is it possible to move all the html tags to the template and see pure java code here? Same for the next change.

Reply:
No, the request content are dynamic. Template only has static place holders.

Reply:
Now the parameter table will be removed. If the request content's key are fixed, we can consider to move them into template and leave here to pure java codes.

Reply:
@epwinchell Please remove https://github.com/RedHatInsights/approval-process/pull/51/files#diff-b621204f4af974b72aac40edf046513fL70 to fix the compilation error